### PR TITLE
Try and improve fit, and add tables

### PR DESCRIPTION
**Pull request by conig**
I have started converting factors to numerics #53
lavaan was complaining when I tried to make factors covary with things, and then I became worried that some factor levels might be ambigous (the wrong way round etc)

making everything e.g. indig [0 , 1] is way safer than as factors.

### CODE REVIEW

**Comment by conig** [comment]
@tarensanders i was being a bit flippant with rebases and managed to merge my PR onto the table PR 🤷
so this is a 2 for 1.

File: R/modelling/fit_lgcm.R [hunk 1]
```diff
@@ -10,68 +10,53 @@
...
     covariates <- glue::glue("{covariates} + fastingtime_w6.5")
   }
 
   model <- glue::glue(
     # nolint start
     "
-    # LGCM
-    intercept =~ 1 * st_total_w3_scaled + 1 * st_total_w4_scaled + 1 * st_total_w5_scaled + 1 * st_total_w6_scaled
-    slope =~ 0 * st_total_w3_scaled + 1 * st_total_w4_scaled + 2 * st_total_w5_scaled + 3 * st_total_w6_scaled
+    # lgcm
```
Comment (tarensanders) [inline R/modelling/fit_lgcm.R:22]: lol what do you have against capital letters?

File: R/modelling/fit_lgcm.R [hunk 2]
```diff
@@ -10,68 +10,53 @@
...
+    {outcome} ~ accmvpa_w6.5_scaled + accsed_w6.5_scaled"
   )
+
   model_outputs <- list()
 
   model_outputs$lgcm_fit <- lavaan::growth(
     model,
     data = transformed_data, missing = "fiml"
   )
+  mi <- lavaan::modindices(model_outputs$lgcm_fit)
```
Comment (tarensanders) [inline R/modelling/fit_lgcm.R:56]: imma merge it anyway, but we can delete this

**Review by tarensanders — APPROVED** [review]

**Pull request merged**